data/core/units.cfg

```diff
@@ -643,7 +667,7 @@ The life span of the wose is unknown, although the most ancient members of this
             castle=60
             cave=80
             frozen=70
-            fungus=-80
+            fungus=80
```

Comment:
This seems like an unintentional change.

fungus got the `-` recently, you probably had an older checkout of the wesnoth repo, before that change, and somehow it ended up here.

Reply:
or rather, it got the `-` in master, but not in `1.14`

Reply:
Yeah, I just noticed that and corrected. I will push it later.